lib/services/alexa/request/lifecycle/update.ts

```diff
@@ -15,11 +17,24 @@ const update = async (runtime: AlexaRuntime): Promise<void> => {
     else if (repeatNumber < 100) repeat = RepeatType.DIALOG;
     storage.set(S.REPEAT, repeat);
   }
-
   turn.set(T.REQUEST, runtime.getRequest());
   variables.set(V.TIMESTAMP, Math.floor(Date.now() / 1000));
 
   await runtime.update();
+
+  let request: InteractRequestType = InteractRequestType.REQUEST;
+  if (input?.requestEnvelope?.request?.type === 'LaunchRequest') {
+    request = InteractRequestType.LAUNCH;
+  }
+  // Track response on analytics system
+  runtime.services.analyticsClient.track(
+    runtime.getVersionID(),
+    Event.INTERACT,
+    request,
```

Comment:
```suggestion
    input?.requestEnvelope?.request?.type === 'LaunchRequest' ? InteractRequestType.LAUNCH : InteractRequestType.REQUEST,
```
we could do this inline as the `request` var is never being reused besides here

Reply:
done